compass/ocean/tests/internal_wave/forward.py

```diff
@@ -50,6 +50,11 @@ def __init__(self, test_case, name='forward', subdir=None, ntasks=1,
             options = {'config_mom_del2': '{}'.format(nu)}
             self.add_namelist_options(options)
 
+        if vlr:
+            # turn vertical Lagrangian-remapping on
+            self.add_namelist_options({
+                'config_vert_advection_method': '"remap"'})
```

Comment:
I could be wrong but I think we always use single-quotes in namelist files.
```suggestion
                'config_vert_advection_method': "'remap'"})
```

Reply:
Thankfully, this issue goes away with the yaml files in polaris.